firedrake/slate/slac/kernel_builder.py

```diff
@@ -766,8 +766,9 @@ def __init__(self, coeffs):
 
 
 class IndexCreator(object):
-    inames = OrderedDict()  # pym variable -> extent
-    namer = UniqueNameGenerator(forced_prefix="i_")
+    def __init__(self):
+        self.inames = OrderedDict()  # pym variable -> extent
+        self.namer = UniqueNameGenerator(forced_prefix="i_")
```

Comment:
This change is already in trunk, some merge/rebase confusion?

Reply:
Fixed.